src/main/java/org/example/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private int addCalledalledCount = 0;
```

Comment:
addCalledalledCount is can be renamed as addCallCount or some other meaningful name